css/_typography.scss

```diff
@@ -9,6 +9,35 @@
     font-size: 36px;
   }
 }
+.title-dark {
+  font-family: $font-main;
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 1.25;
+  text-transform: uppercase;
+  color: $color-text-main;
+  @include media-breakpoint-up(md) {
+    font-size: 24px;
+  }
+}
+.card-caption {
+  font-family: $font-main;
+  font-weight: 600;
+  font-size: 14px;
+  line-height: 1.3;
+  color: $color-text-light;
+  text-transform: capitalize;
+}
+.card-list__caption {
```

Comment:
Maybe it will be better to move this element to other card-list elements. I mean to _layout.scss file